extension/src/ALObject/ALParser.ts

```diff
@@ -358,6 +358,14 @@ export function matchALControl(
       );
       control.xliffTokenType = XliffTokenType.control;
       break;
+    case "systempart":
+      control = new ALPagePart(
+        ALControlType.systemPart,
+        alControlResultFiltered[2],
+        alControlResultFiltered[3]
```

Comment:
I don't really like these indexes. But I guess options are scarce?

Reply:
Yeah, I know. I will refactor this to use groups one day. (See how I did for Enum Values the other day)
It's just a too big thing to do today :)